elide-integration-tests/src/test/java/com/yahoo/elide/tests/AnyPolymorphismIT.java

```diff
@@ -0,0 +1,269 @@
+                .statusCode(HttpStatus.SC_OK)
+                .body(includedType, equalTo(smartphoneType),
+                        includedId, equalTo(one),
+                        "included[0].attributes.type", equalTo("android"),
+                        includedSize, equalTo(1),
+                        "included[0].attributes.operatingSystem", equalTo("some dessert"));
+
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get(propertyPath)
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("data.size()", equalTo(2));
+    }
+
+    @Test
+    public void testAnyupdate() {
+        String id = RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .body(jsonParser.getJson(tractorAsPropertyFile))
+                .post(propertyPath)
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_CREATED)
+                .body(relationshipType, equalTo(tractorType))
+                .extract()
+                .path(idPath);
+
+        String prefix = "{\"data\": {\"id\": \"" + id;
+
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .body(prefix + "\", \"type\": \"property\", \"relationships\": {\"myStuff\": {\"data\": {\"type\": \"smartphone\", \"id\": \"1\"}}}}}")
+                .patch(propertyPath + sep + id)
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_NO_CONTENT);
+
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get(propertyPath + sep + id)
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body(relationshipType, equalTo(smartphoneType));
+
+        //delete relation
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .body(prefix + "\", \"type\": \"property\", \"relationships\": {\"myStuff\": {\"data\": null}}}}")
+                .patch(propertyPath + sep + id)
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_NO_CONTENT);
+
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get(propertyPath + sep + id)
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("data.relationships.myStuff.data", equalTo(null));
+    }
+
```

Comment:
I'm curious to see what would happen if you tried to query /device

Reply:
It returns a 404 like it should.  added test to show this.